filebeat/fileset/config.go

```diff
@@ -0,0 +1,17 @@
+package fileset
+
+// ModuleConfig contains the configuration file options for a module
+type ModuleConfig struct {
+	Module   string                    `config:"module"     validate:"required"`
+	Enabled  *bool                     `config:"enabled"`
+	Filesets map[string]*FilesetConfig `config:"filesets"`
```

Comment:
Interesting. Is there a reason you prefix the fileset config options with `fileset`. Namespace conflicts? The convention in metricbeat is to just use the metricset name, so above it would be `error.enabled`.

An other option about would be to use `filesets: ["access"]` if you want only one. If `filesets` is not around, all sets are loaded. I agree this makes sense in the filebeat module case. But this change would require the previous change, as otherwise `filesets` would have 2 meanings.